File: src/main/java/dev/sky_lock/pocketlifevehicle/gui/CarUtilMenu.java
package dev.sky_lock.pocketlifevehicle.gui;

import dev.sky_lock.menu.InventoryMenu;
import dev.sky_lock.pocketlifevehicle.vehicle.Car;
import dev.sky_lock.pocketlifevehicle.gui.contents.CarUtilContents;
import org.bukkit.entity.Player;

/**
 * @author sky_lock
 */

public class CarUtilMenu extends InventoryMenu {

    public CarUtilMenu(Player holder, Car car) {
        super("ユーティリティ", Type.BIG, holder);
        addContents(new CarUtilContents(car));
    }
}

File: src/main/kotlin/dev/sky_lock/pocketlifevehicle/gui/CarUtilMenu.kt
package dev.sky_lock.pocketlifevehicle.gui

import dev.sky_lock.menu.InventoryMenu
import dev.sky_lock.pocketlifevehicle.gui.contents.CarUtilContents
import dev.sky_lock.pocketlifevehicle.vehicle.Vehicle
import org.bukkit.entity.Player

/**
 * @author sky_lock
 */
class CarUtilMenu(holder: Player, vehicle: Vehicle) : InventoryMenu("ユーティリティ", Type.BIG, holder) {
    init {
        addContents(CarUtilContents(vehicle))
    }
}